server/src/entities/response.ts

```diff
@@ -0,0 +1,15 @@
+import { IAnswer } from "./answer";
+import { IUser } from "./user";
+
+interface IScore {
+  score: number;
+  notes: string;
+  user: IUser;
+}
+
+export interface IResponse {
+  id: string;
+  respondent: string;
+  answers: [IAnswer];
+  scores: [IScore];
```

Comment:
Will the fact that these fields are ObjectIds unless populated affect anything?

Reply:
If it becomes an issue we can make them nullable, or change the type to `any`, or make sure we populate, or use a union type of some sort (`ObjectID | [IAnswer]`?)